app/views/booking_requests/customer.html.erb

```diff
@@ -35,5 +35,5 @@ class="emphasize" style="font-size: 24px;"><%=
 
 <p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
 0;font-size: 16px;line-height: 1.315789474;">To cancel or change your
-appointment, or if any of the information above isn't correct, call 0800 138
-3944 or reply to this email.</p>
+appointment, or if any of the information above isn't correct, call
+ <%= @booking_location.online_booking_twilio_number %> or reply to this email.</p>
```

Comment:
Do we want to use the Twilio number here?

As this could distort the stats the Twilio number is designed to capture i.e. number of appointments coming from the website via phone channel.